src/xdocs/config_design.xml

```diff
@@ -1516,47 +1678,87 @@ public class ImmutableClass
+
+ protected String field2;  // violation, protected visibility is not allowed
+
+ public int field3 = 42;  // violation, not static final, immutable,
+                          // nor matching the pattern
+ public long serialVersionUID = 1L;
+ public static final int field4 = 42;
+
+ public final int field5 = 42;
+ public final java.lang.String notes;
+
+ public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;();  // violation, HashSet is mutable
+ public final ImmutableSet&lt;String&gt; mySet2;
+ public final ImmutableMap&lt;String, Object&gt; objects1; // violation
+                                                     // 'Object' still considered as mutable
+
+ @com.annotation.CustomAnnotation
+ String annotatedString;      // violation
+
+ @CustomAnnotation
+ String shortCustomAnnotated; // violation
+
+ @com.google.common.annotations.VisibleForTesting
+ public String testString = "";
 }
         </source>
         <p>
           To configure the Check passing fields annotated with @com.annotation.CustomAnnotation:
         </p>
         <source>
 &lt;module name=&quot;VisibilityModifier&quot;&gt;
-  &lt;property name=&quot;ignoreAnnotationCanonicalNames&quot; value=
-  &quot;com.annotation.CustomAnnotation&quot;/&gt;
+  &lt;property name=&quot;ignoreAnnotationCanonicalNames&quot;
+        value=&quot;com.annotation.CustomAnnotation&quot;/&gt;
 &lt;/module&gt;
         </source>
         <p>
           Example of allowed field:
         </p>
         <source>
-class SomeClass
-{
-  @com.annotation.CustomAnnotation
-  String annotatedString; // no warning
-  @CustomAnnotation
-  String shortCustomAnnotated; // no warning
+public final class MyClass {
+ private int myPrivateField1;
+
+ int field1;               // violation, must have a visibility modifier
+
+ protected String field2;  // violation, protected visibility is not allowed
+
+ public int field3 = 42;  // violation, not static final, immutable,
+                          // nor matching the pattern
+ public long serialVersionUID = 1L;
+ public static final int field4 = 42;
+
+ public final int field5 = 42;          // violation, public immutable fields are not allowed
+ public final java.lang.String notes;   // violation, public immutable fields are not allowed
+
+ public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;();  // violation, HashSet is mutable
+ public final ImmutableSet&lt;String&gt; mySet2;           // violation
+                                                     // immutable type not in config
+ public final ImmutableMap&lt;String, Object&gt; objects1; // violation
+                                                     // immutable type not in config
+
+ @com.annotation.CustomAnnotation
+ String annotatedString;
+
+ @CustomAnnotation
+ String shortCustomAnnotated;
+
+ @com.google.common.annotations.VisibleForTesting
+ public String testString = "";                             // violation
```

Comment:
Why this `// violation` comment is indented so far right, it is not aligning with any other comment, please reduce indentation to align it vertically with previous violation comments.

Reply:
done

Reply:
Resolved.